src/Services/Blamable/HostnetBlamableProvider.php

```diff
@@ -0,0 +1,55 @@
+<?php
+namespace Hostnet\Bundle\EntityTrackerBundle\Services\Blamable;
+
+use Hostnet\Component\EntityBlamable\Provider\BlamableProviderInterface;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+
+/**
+ * Provides the logged username and the current time for the entities that will be using the
+ * blamable component.
+ *
+ * @author Eunice Valdez <evaldez@hostnet.nl>
+ */
+class HostnetBlamableProvider implements BlamableProviderInterface
```

Comment:
Code should not be aware of the service concept. Can you move this class to the following namespace: `Hostnet\Bundle\EntityTrackerBundle\Blamable`?

Can you rename this class to `DefaultBlamableProvider`?